EPM VSAT documentation (#924)
SkylineCommunications/dataminer-docs: user-guide/Standard_Apps/EPM/EPM_VSAT/VSAT_EPM_Topology.md

---
uid: VSAT_EPM_Topology
---

# EPM VSAT topology

The EPM topology navigation menu can be used to access the collector elements' main KPIs.

> ![Topology](~/user-guide/images/VSAT_EPM_Topology.png)

Reviewer: Maybe we should add the DataMiner version and solution version for this screenshot, possibly in a figure caption below it, which can be configured like this: 
```suggestion
![Topology](~/user-guide/images/VSAT_EPM_Topology.png)<br>
*figure caption*
```

This way, if the screenshot gets outdated, users will know why it's different from what they see in their more recent system. The same could also apply for other update-sensitive screenshots in this PR.

## Topology chain

> Below you can find an overview of the different chains the VSAT EPM Solution offers and can be accessed in the *Topology* tree and their subsequent entities available at each branch.

Reviewer: ```suggestion
Below you can find an overview of the different chains the VSAT EPM Solution offers, which can be accessed in the *Topology* tree, and the subsequent entities available in each branch.
```

![Topology Tree](~/user-guide/images/Topology_Tree.png)

- **Network**:

  - Network
  - Teleport
  - Hub Network
  - Hub Return
  - Hub Forward
  - Circuit

- **Service**:

  - Network
  - Customer
  - Customer/NMS
  - Circuit

- **NMS**:

  - Network
  - NMS
  - Hub Network
  - Hub Return
  - Hub Forward
  - Circuit

- **Map**:

  - Network
  - NMS
  - Customer
  - Customer/NMS
  - Circuit

- **Quick**:

  - Type
  - Network

- **Infrastructure**:

  - NMS
  - PP Server
  - Chassis
  - Linecards

- **Configuration**

## Data pages

> Each entity in the Topology tree has an associated data page where relevant KPI and aggregations can be shown. This can be accessed by clicking the arrow to the right of the circuit filter in the Topology tree. Below, you can find an example of a Circuit Data page.

Reviewer: ```suggestion
Each entity in the Topology tree has an associated data page where relevant KPIs and aggregation results can be shown. You can access this by clicking the arrow to the right of the circuit filter in the Topology tree. Below, you can find an example of a Circuit Data page.
```
